### PR TITLE
feat: [UEPR-90] Added new section to the ideas page

### PR DESCRIPTION
**Pull request by MiroslavDionisiev**
### Jira Ticket:
[UEPR-90](https://scratchfoundation.atlassian.net/browse/UEPR-90)

### Changes:

Added new sections to ideas page, removed old ones, changed the page component from class to functional one.

### Test Coverage:

_Please show how you have added tests to cover your changes or describe how you have tested the changes (include a screenshot if possible)._
image
image
image

[UEPR-90]: https://scratchfoundation.atlassian.net/browse/UEPR-90?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/views/ideas/l10n.json [hunk 1]
```diff
@@ -4,7 +4,11 @@
     "ideas.headerButtonMessage": "Choose a tutorial",
     "ideas.gettingStartedTitle": "Getting Started",
     "ideas.gettingStartedText": "New to Scratch? Try the Getting Started tutorial.",
```
Comment (KManolov3) [inline src/views/ideas/l10n.json:6]: Can we remove unused translations from here, if there are any

File: src/views/ideas/ideas.scss [hunk 2]
```diff
@@ -17,127 +17,133 @@ $base-bg: $ui-white;
...
+    font-family: "Helvetica Neue", "Helvetica", Arial, sans-serif;
+    font-weight: 700;
 }
 
-.tips-info-body {
-    text-align: left;
-}
+.tips-button {
+    display: flex;
+    flex-direction: row;
```
Comment (KManolov3) [inline src/views/ideas/ideas.scss:132]: Isn't row the default direction?

File: src/views/ideas/ideas.scss [hunk 3]
```diff
@@ -17,127 +17,133 @@ $base-bg: $ui-white;
...
-}
-
-.tips-banner-image {
-    max-width: calc(100% - 2rem);
-}
-
-.banner-button {
-    background-color: $ui-white;
-    color: $ui-aqua;
-    font-size: 1rem;
```
Comment (mxmou) [inline src/views/ideas/ideas.scss:41]: Were these styles intentionally removed? `ideas.jsx` still references this class (line 104).

Comment (MiroslavDionisiev) [reply src/views/ideas/ideas.scss:41]: Yes, the intent for this task was to redesign the header as well but the content team delayed providing some links so we will handle it separately.

**Pull request merged**